docs/collectors.md

```diff
@@ -0,0 +1,108 @@
+# Collectors
+
+In order to use Rode to drive Automated Governance, we need a strategy for _collecting_
+metadata from events in our toolchain. In Rode, **collectors** are responsible for
+recieving events from exte rnal systems and converting them into
+[occurrences](https://github.com/grafeas/grafeas/blob/master/docs/grafeas_concepts.md#occurrences).
+These occurrences can be used to track the history of events for a particular
+artifact. These collectors are to be built as individual services and can be
+maintained independently of the central Rode API. This enables developers to build
+collector services with a language of their choosing([_provided it supports gRPC_](https://grpc.io/docs/languages/))
+and add new functionality at their own pace.
+
+## Development
+
+As mentioned previously, collectors are independent services and can be built with
+any language or framework that has support for gRPC. Collectors typically have two
+primary responsibilities:
+
+- Listening for incoming event metadata associated with artifacts _for a single service_
+- Publishing structured data to the Rode API in the form of _occurrences_
```

Comment:
Should we indicate anywhere that you can also go back and retrieve more info if you want

Reply:
Good idea @aalsabag, just added a bit more info. Let me know what you think

Reply:
I like that last paragraph. Nice job @gesparza3 !